controllers/ibu_controller.go

```diff
@@ -191,6 +199,12 @@ func (r *ImageBasedUpgradeReconciler) handleStage(ctx context.Context, ibu *lcav
 	case lcav1alpha1.Stages.Rollback:
 		nextReconcile, err = r.handleRollback(ctx, ibu)
 	}
+
```

Comment:
If there is any hander error, it gets returned to k8s and any update made here gets discarded. K8s basically calls the reconcile again with the original version of the CR.

Reply:
Reverted